iiod-responder.c

```diff
@@ -670,14 +670,14 @@ iiod_responder_create(const struct iiod_responder_ops *ops, void *d)
 	      goto err_free_lock;
 
 	priv->write_task = iio_task_create(iiod_responder_write, priv,
-					   "iiod-responder-writer-task");
+					   "writer-task");
 	err = iio_err(priv->write_task);
 	if (err)
 		goto err_free_io;
 
 	if (!NO_THREADS) {
 		priv->read_thrd = iio_thrd_create(iiod_responder_reader_thrd, priv,
-						  "iiod-responder-reader-thd");
+						  "reader-thd");
```

Comment:
Given the length constraint, I suggest removing "-thd" since it's already clear that we're naming threads. Instead, those four characters could be used to better describe the thread or distinguish it from others in relevant situations.

Reply:
Hmm, it does make sense. I'll see if i can thing on something better... I'm reworking things a bit as I just realized (stupid me) that we do need a "pure" define in here and not something that evaluates to 0 or 1. If the API is not available and place inside an if(0) it will still fail to compile. Dead code removal happens in a later point in time...

Reply:
Hmm thinking a bit more I can replace it for something like job (we do sabe 1 char). Or "client-reader" but here your argument also applies (we already now it's the client). One thing that could be beneficial would be to differentiate between the clients created on behave of buffer and the main client. But the trouble for that seemed not worth it for me.